src/scep/Client/certificate.py

```diff
@@ -3,12 +3,22 @@
 
 from cryptography import x509 as crypto_x509
 from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import serialization
 
 from .privatekey import PrivateKey
 from .publickey import PublicKey
 
 
 class Certificate:
+    
+    @classmethod
+    def get_x509_certificate(cls, certificate_raw_file, get_public_bytes=False):
```

Comment:
how is this different than `from_pem_file()` below?

Reply:
This function is not being used anywhere in the code, I'll remove it.